### DISCUSSION TITLE
Shall there be an option to reuse existing packages/modules from repositories ?

### DISCUSSION THREAD

**Discussion by saurshaz** [beakerbrowser/beaker#1783] [General]
```
// ## hyper://6cfb68e0e85ddadeed60f8c497082c67852d616b25397d89a63aa042b92a1a61/play.mjs
  const url = await get('https://unpkg.com/lodash@4.17.19/lodash.js', 'text/javascript');
  console.log((url));
  addScriptToPage(url);
```

I was writing something like this, which adds a hyper link which has `lodash` code in a blob link. But is the right quest ?? Will it end up being managing entire dependency graph ? Thought welcome

**Comment by pfrazee** [comment]
Why do it that way instead of `<script src="">` or the ES module `import from`? Is the code in commonjs?

**Reply by saurshaz** [reply]
yes,  talking of importing modules which are in npmjs repo/unpkg.com . 

I think that'll encourage people to bring/create more and more projects in beaker ecosystem.

**Reply by pfrazee** [reply]
Ah yeah. Strongly agree. We're iterating our way towards how to handle modules.
